recipes/py-triangle/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "py-triangle" %}
+{% set version = "v20220202" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   - url: https://github.com/drufat/triangle/archive/refs/tags/{{ version }}.tar.gz
+     sha256: e975631b82538a9c008a34e415e52f1a60969f98d819fd7ca6ad7906352e33d1
+   - git_url: https://github.com/drufat/triangle-c
+     git_rev: 8b9e1046e5cddab1298d3204f10c93665836cf99 # Keep that in sync with triangle python package 
+     git_depth: 1
+     folder: c
```

Comment:
Please link against the existing triangle package instead of revending.

https://github.com/conda-forge/triangle-feedstock

Reply:
I would love too. 
Unfortunately, the python bindings rely on this patched version of the triangle library: https://github.com/drufat/triangle-c

Reply:
OK. It seems like you have tomplied with license requirements.